workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java

```diff
@@ -78,7 +78,10 @@ public void shutdownExecutorService() {
 
     @Override
     public void onWorkflowCompleted(WorkflowModel workflow) {
-        LOGGER.info("Archiving workflow {} on completion ", workflow.getWorkflowId());
+        LOGGER.info("Archiving workflow. Workflow Name : {} Workflow Id : {} Workflow Status : {} ",
```

Comment:
Should we have each workflow's archival as INFO or DEBUG log?

Reply:
We are printing only error msgs in conductor so these logs will not get printed anyway. I have changed the log level to DEBUG in S3 archival class which we will be using. I will change these too.

Reply:
Done